ui/rla_ui.js

```diff
@@ -8,968 +8,996 @@
+			confirmationDiv.parentNode.removeChild(confirmationDiv);
+		};
+
+		confirmationDiv.appendChild(candidateSelectionList(ballotType, interpretationJSON, optionalListOfContests));
+		confirmationDiv.appendChild(rejectButton);
+		confirmationDiv.appendChild(confirmButton);
+		return confirmationDiv;
+	}
+
+	function candidateSelectionList(ballotType, interpretationJSON) {
+		var resultList = [];
+		/*
+		   for (var contestId in interpretationJSON['contests']) {
+		      resultList.push(contestId + ': ' + interpretationJSON['contests'][contestId]);
+		   }
+		*/
+		conductorState['all_contests'][ballotType].forEach(function (contest) {
+			if (typeof (interpretationJSON['contests'][contest.id]) != 'undefined') {
+				resultList.push(contest.title + ': ' + prettifyChoice(interpretationJSON['contests'][contest.id]));
+			}
+		});
+		return buildOrderedList(resultList);
+	};
+
+	function prettifyChoice(choiceName) {
+		// the entry with "DEM" should be superfluous, see what follows
+		var choiceMap = {
+			'DEM David N. Cicilline (13215)': 'David N. Cicilline',
+			'David N. Cicilline (13215)': 'David N. Cicilline',
+			'Write-in': 'Write-in candidate',
+			'undervote': 'No selection (undervote)',
+			'overvote': 'Overvote',
+			'Representative in Congress District 1 (13213) ': 'Rep. in Congress Dist. 1',
+			'1. RHODE ISLAND SCHOOL BUILDINGS - $250,000,000': 'Question 1 (schools)',
+			'2. HIGHER EDUCATION FACILITIES - $70,000,000': 'Question 2 (higher ed)',
+			'3. GREEN ECONOMY AND CLEAN WATER - $47,300,000': 'Question 3 (green econ)',
+		};
+		if (['DEM ', 'REP ', 'MOD ', 'Com ', 'Ind '].includes(choiceName.substring(0, 4))) {
+			choiceName = choiceName.substring(4);
+		};
+		return choiceMap[choiceName] || choiceName;
+	};
+
+	function newRaceCheckbox(ballot_id, race_id, race_title, race_choices) {
+		var div, ul;
+		div = newElem('div');
+		div.innerText = race_title;
+		div.classList.add('raceDiv');
+
+		ul = newElem('ul');
+		ul.setAttribute('style', 'list-style-type: none');
+		div.appendChild(ul);
+		race_choices.concat(['Write-in', 'undervote', 'overvote']).forEach(function (choice, i) {
+			if (!(race_id.includes('issue') && choice.includes('Write-in'))) {
+				//race_choices.forEach(function(choice,i) {
+				var checkbox, label, li;
+				checkbox = newElem('input');
+				checkbox.type = 'radio'; // 'checkbox';
+				checkbox.name = contestCheckboxName(ballot_id, race_id);
+				// TODO: better-guarantee that these are unique
+				// (Almost definitely not a concern for RI):
+				checkbox.id = ballot_id + '#' + race_id + '#' + race_title + '#' + i;
+				checkbox.classList.add('choiceCheckbox');
+				checkbox.value = choice;
+
+				label = newElem('label');
+				label.innerHTML = prettifyChoice(choice); // again, careful!
+				label.setAttribute('for', checkbox.id);
+				label.classList.add('choiceCheckboxLabel');
+
+				li = newElem('li');
+
+				li.appendChild(checkbox);
+				li.appendChild(label);
+				ul
```

Comment:
Can use `getById` instead of `document.getElementById`

https://github.com/ElectionAuditWare/audit-conductor/blob/3de6dca06854a035072a8fab49a984cad7d52bb0/ui/rla_ui.js#L56